setup.py

```diff
@@ -5,7 +5,12 @@
 )
 
 DESCRIPTION = 'Snowflake Telemetry Python'
-LONG_DESCRIPTION = 'This package provides a set of telemetry APIs for use in Snowflake'
+LONG_DESCRIPTION = """This package provides a set of telemetry APIs for developers building on the Snowflake platform.
+
+Snowflake Documentation is available at: https://docs.snowflake.com/
```

Comment:
Should this directly point to the Telemetry Python documentation to make it more easy to find the API usage?

Reply:
I updated the link to point to https://docs.snowflake.com/en/developer-guide/logging-tracing/tracing-python